src/apps/unagii/common/types.ts

```diff
@@ -0,0 +1,5 @@
+export type UnagiiTokenDefinition = {
+  address: string;
+  underlyingTokenAddress: string;
+  vaultManagerAddress: string;
+};
```

Comment:
Move this into the `src/apps/unagii/common/unagii.vault-token-fetcher.ts` class directly, see https://kentcdodds.com/blog/colocation for some good advice re: colocation of code.